<?xml version="1.0"?>
<launch>

<!-- =========================================================== --> 
<!--common args -->

    <arg name="respawn" default="false" />
    <arg name="debug" default="false" />
    <arg unless="$(arg debug)" name="launch_prefix" value="" />
    <arg     if="$(arg debug)" name="launch_prefix" value="gdb -ex run --args" />
        
    




<!-- =========================================================== -->
<!-- display -->
<!-- =========================================================== -->
  
  <node name="image_view_pylon" pkg="image_view" type="image_view" output="screen">
      <param name="image_transport" value="compressed"/>
      <remap from="image" to="/BW_camera_node/image_raw"/>
  </node>

    
</launch>
